docs/source/examples/Natural and artificial perturbations.mystnb

```diff
@@ -81,18 +88,15 @@ def f(t0, state, k):
     return du_kep + du_ad
 
 
-rr = propagate(
-    orbit,
-    tofs,
-    method=cowell,
-    f=f,
-)
+rr, _ = orbit.to_ephem(
+    EpochsArray(orbit.epoch + tofs, method=CowellPropagator(f=f)),
+).rv()
 ```
 
-```{code-cell} ipython3
+```{code-cell}
 plt.ylabel("h(t)")
 plt.xlabel("t, days")
-plt.plot(tofs.value, rr.norm() - Earth.R)
+plt.plot(tofs.value, norm(rr, axis=1) - Earth.R)
```

Comment:
Since `Ephem.rv()` returns `Quantity` objects rather than `CartesianRepresentation` objects, I extended our `poliastro.util.norm` function to work with `axis=1` too.